src/pivotal-ui/components/iconography/iconography.scss

```diff
@@ -13,47 +13,87 @@ categories:
+<svg class="icon"><use xlink:href="#i-download"/></svg>
 npm install @npmcorp/pui-css-iconography --save
-</i>
 </code>
 
 
-We use [Font Awesome](http://fortawesome.github.io/Font-Awesome/icons/) for scalable vector icons that can be customized - size, color, drop shadow,
-and anything that can be done with CSS. These icons will always match the font color.
+We use [Glyphish](http://glyphish.comhttp://www.glyphish.com) for scalable icons. They have been packaged up as [@npmcorp/glyphish] for use, and inserting them as SVG `<symbol>` allows us to use the same icon multiple times with only a minimal bit of code for further instances.
+
+These icons will always match the font color.
 
 Here are some commonly used icons:
 
-Button                                             | Class                  | Description
--------------------------------------------------- | ---------------------- | -----------
-<i class="fa fa-plus"></i>                         | `.fa.fa-plus`          | Renders our plus icon
-<i class="fa fa-user"></i>                         | `.fa.fa-user`          | Renders our person icon
-<i class="fa fa-sort-down"></i>                    | `.fa.fa-sort-down`     | Renders our triangle icon
-<i class="fa fa-cog"></i>                          | `.fa.fa-cog`           | Renders our gear icon
-<i class="fa fa-times"></i>                        | `.fa.fa-times`         | Renders our 'x' icon
-<i class="fa fa-check"></i>                        | `.fa.fa-check`         | Renders our check icon
-<i class="fa fa-chevron-right"></i>                | `.fa.fa-chevron-right` | Renders our right chevron icon
-<i class="fa fa-chevron-up"></i>                   | `.fa.fa-chevron-up`    | Renders our down chevron icon
-<i class="fa fa-chevron-down"></i>                 | `.fa.fa-chevron-down`  | Renders our down chevron icon
-<i class="fa fa-windows"></i>                      | `.fa.fa-windows`       | Renders our Windows icon
-<i class="fa fa-apple"></i>                        | `.fa.fa-apple`         | Renders our Apple icon
-<i class="fa fa-linux"></i>                        | `.fa.fa-linux`         | Renders our Linux icon
-<i class="fa fa-pencil"></i>                       | `.fa.fa-pencil`        | Renders our edit icon
-<i class="fa fa-plus-circle"></i>                  | `.fa.fa-plus-circle`   | Renders our second add icon
-<i class="fa fa-trash-o"></i>                      | `.fa.fa-trash-o`       | Renders our delete icon
-<i class="fa fa-square"></i>                       | `.fa.fa-square`        | Renders our square icon
-<i class="fa fa-repeat"></i>                       | `.fa.fa-repeat`        | Renders our refresh/restart icon
-<i class="fa fa-play"></i>                         | `.fa.fa-play`          | Renders our start icon
-<i class="fa fa-download"></i>                     | `.fa.fa-download`      | Renders our download icon
+Button                                                  | Filename
```

Comment:
Doh, thank you. That was the original heading and I never changed it. (doesn't make any sense there either!)